src/tests/plugins/test_sendmail.py

```diff
@@ -94,5 +93,84 @@ def test_sendmail_one_message(logged_in_client, sendmail_url, event, order, expe
     url = sendmail_url + 'history/'
     response = logged_in_client.get(url)
 
-    assert response.status_code == expected
+    assert response.status_code == 200
     assert 'Test subject' in response.rendered_content
+
+
+@pytest.mark.django_db
+def test_sendmail_email_not_sent_to_order(logged_in_client, sendmail_url, event, order):
```

Comment:
> and so far Raphael doesn't seem to mind

I can't see the day coming on which I would mind ;)

Flavia, I'm gonna try to review this tonight, let's see if I actually get around to do it :D